validator.go

```diff
@@ -27,6 +28,77 @@ type messageValidator struct {
 	useTransportMessageAttributes bool
 }
 
+func (v *messageValidator) getPayloadandAttributes(message *Message, runWithTransportMessageAttributes *bool) ([]byte, map[string]string, error) {
```

Comment:
don't love the name or the verbosity of this param but don't have a better suggestion other than `overrideUseTransportMsgAttrs`